### PR TITLE
Update WE2E documentation

### PR DESCRIPTION
**Pull request by gsketefian**
## DESCRIPTION OF CHANGES: 
This updates the documentation for how to use the WE2E testing system.

## TESTS CONDUCTED: 
Compiled the documentation using `sphinx-build` and viewed the resulting html.

## DEPENDENCIES:
PR #[745](https://github.com/ufs-community/regional_workflow/pull/745) in regional_workflow.

## CONTRIBUTORS (optional): 
@gspetro

### CODE REVIEW

**Review by chan-hoo — APPROVED** [review]

**Comment by gsketefian** [comment]
@gspetro I just updated the chapter on WE2E testing.  Do you mind taking a look?

File: docs/UsersGuide/source/WE2Etests.rst
Comment (gspetro-NOAA) [inline docs/UsersGuide/source/WE2Etests.rst:29]: @jkbk2004 @EdwardSnyder-NOAA Aren't you guys working on SRW App regression tests? Can we add anything here?

**Review by JeffBeck-NOAA — APPROVED** [review]

File: docs/UsersGuide/source/WE2Etests.rst
Comment (gspetro-NOAA) [inline docs/UsersGuide/source/WE2Etests.rst:415]: @fossell Is this mention (lines 411-416) of inline post okay by your team?

File: docs/UsersGuide/source/WE2Etests.rst
Comment (EdwardSnyder-NOAA) [reply docs/UsersGuide/source/WE2Etests.rst:29]: I have been working on WE2E testing for the SRW App, but I still read over the changes and see nothing to add.

File: docs/UsersGuide/source/WE2Etests.rst
Comment (fossell) [reply docs/UsersGuide/source/WE2Etests.rst:415]: @gspetro-NOAA - It seems that inline post is just being used as an example here for another purpose (creating new tests), but it does seem to imply that inline post could be a supported feature should a user want to turn it on. I think we need to be clear up front about whether or not it is or isn't supported, instead of having it sporadically appear in different parts of the documentation. The feature is functioning for regional applications now, but earlier this year a decision was made to go with offline only for SRW. Let me discuss with other UPP code managers and get a final decision on this so we don't have to deal with word-smithing documentation like this.

Comment (fossell) [reply docs/UsersGuide/source/WE2Etests.rst:415]: @gspetro-NOAA - After discussion with EMC code managers, I think this sort of documentation is ok to keep. The feature works from a technical standpoint, so as long as those in charge of supporting this release are ok with this mention it's fine.

Comment (gsketefian) [reply docs/UsersGuide/source/WE2Etests.rst:415]: @gspetro-NOAA I'll wait to get the ok from you to @fossell's response before merging.  Thanks.

Comment (gspetro-NOAA) [reply docs/UsersGuide/source/WE2Etests.rst:415]: @gsketefian You're all set to merge!

Comment (gsketefian) [reply docs/UsersGuide/source/WE2Etests.rst:415]: @gspetro-NOAA Thanks!

**Pull request merged**
